weasel/_util.py

```diff
@@ -1,30 +1,35 @@
 # code initially copied from spacy/cli/_util.py
 
-from typing import Dict, Any, Union, List, Optional, Tuple, Iterable
-from typing import TYPE_CHECKING
-import sys
+import hashlib
+import os
 import shutil
+import subprocess
+import sys
+from configparser import InterpolationError
+from contextlib import contextmanager
 from pathlib import Path
-from wasabi import msg
+from typing import TYPE_CHECKING, Any, Dict, Iterable, List, Optional, Tuple, Union
+
 import srsly
-import hashlib
 import typer
 from click import NoSuchOption
 from click.parser import split_arg_string
-from contextlib import contextmanager
 from thinc.api import Config, ConfigValidationError
-from configparser import InterpolationError
-import os
-
-from .util import run_command, make_tempdir, logger
-from .util import is_compatible_version, SimpleFrozenDict, ENV_VARS
-
-from .schemas import ProjectConfigSchema, validate
+from wasabi import msg
 
 from . import about
+from .schemas import ProjectConfigSchema, validate
+from .util import (
+    ENV_VARS,
+    SimpleFrozenDict,
+    is_compatible_version,
+    logger,
+    make_tempdir,
+    run_command,
+)
```

Comment:
Is there a way to get all the sorting/reformatting to disallow multi-line imports? We'd usually prefer something more like:

```suggestion
from .util import ENV_VARS, SimpleFrozenDict, is_compatible_version
from .util import logger, make_tempdir, run_command
```

(It's not that black supports this either, it would just be nice if there were an option here.)

Reply:
Ok, the only option that resembles this is: https://beta.ruff.rs/docs/settings/#force-single-line

```python
from .util import ENV_VARS
from .util import SimpleFrozenDict
from .util import is_compatible_version
from .util import logger
from .util import make_tempdir
from .util import run_command
```

But between the two I prefer the less verbose alternative. Of course we can also just turn off `isort` capabilities.

That being said, I actually prefer multiline imports: they make it more readable and minimises diffs... (mostly reciting the [black gospel](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#how-black-wraps-lines:~:text=If%20a%20data,or%20manual%20configuration.) here)

Reply:
It's true that we've been avoiding multi-line imports in spaCy and related packages up until now, but I personally feel like the advantages of using `isort` (and the multi-line imports like in this PR) outweigh them taking up a few more lines (actually in PyCharm, you can have the imports be collapsed by default so they don't get in the way either)

Reply:
Ah wait: https://pycqa.github.io/isort/docs/configuration/multi_line_output_modes.html#9-vertical-prefix-from-module-import

Reply:
Ok this is not supported by Ruff. We can use `isort` separately - I'd rather we used a single faster tool but I'm biased...

I've pushed a version that uses `isort` to be consistent with other packages.